### PR TITLE
add test multipleswap

### PR DESCRIPTION
**Pull request by wojciech-cichocki**

### CODE REVIEW

File: tests/multiple-swap.spec.ts [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+          pair,
+          amount: swapAmount,
+          estimatedPriceAfterSwap: { v: priceAfterSwap },
+          accountX: user2AccountX,
+          accountY: user2AccountY,
+          owner: user2.publicKey,
+          ...swapParams
+        },
+        user2
+      )
```
Comment (robertg93) [inline tests/multiple-swap.spec.ts:240]: without any checks ?

File: tests/testUtils.ts [hunk 2]
```diff
@@ -297,13 +297,15 @@ export const createTicksFromRange = async (
...
+export const initMarket = async (
   market: Market,
   pairs: Pair[],
   admin: Keypair,
   initTick?: number
 ) => {
-  await market.createState(admin.publicKey, admin)
+  try {
+    await market.createState(admin.publicKey, admin)
+  } catch (e) {}
```
Comment (robertg93) [inline tests/testUtils.ts:308]: why in try catch ?

Comment (wojciech-cichocki) [reply tests/testUtils.ts:308]: In case market have already initialized

File: tests/multiple-swap.spec.ts (hunk 1 above)
Comment (wojciech-cichocki) [reply tests/multiple-swap.spec.ts:240]: will add some

**Pull request merged**